Resources/Locale/en-US/thief/backpack.ftl

```diff
@@ -23,10 +23,11 @@ thief-backpack-category-chameleon-description =
     Disguise as anyone and anything.
 
 thief-backpack-category-tools-name = breacher kit
+#Delta V: C4 replaced with breaching charges and description changed
```

Comment:
using \r\n or something its red on github

Reply:
uhh wdym?

Reply:
Add a space between "#" and "Delta"